website/common/script/fns/randomDrop.js

```diff
@@ -79,8 +79,14 @@ export default function randomDrop (user, options, req = {}, analytics) {
         canDrop: true,
       })));
 
-      user.items.food[drop.key] = user.items.food[drop.key] || 0;
-      user.items.food[drop.key] += 1;
+      user.items.food = {
```

Comment:
only the second one here is necessary to trigger Vue reactivity

Reply:
see previous comment